src/components/common/ExpressionInput.vue

```diff
@@ -108,8 +110,9 @@ export class ExpressionInput extends mixins(Lang) {
    */
   portAutoSuggestContent(): void {
     // override position which came from AutoSuggest original code
+    this.autoSuggestDropdown.style.position = 'relative'
```

Comment:
by default, the dropdown has "absolute" position, and some values on left & top.
The solution is just to get rid of that to have relative position, and then allow flex display to handle a column display https://getbootstrap.com/docs/4.0/utilities/flex/#direction